ballerina/tests/test_services.bal

```diff
@@ -1142,3 +1142,61 @@ service /nullable_inputs on basicListener {
         return account.number;
     }
 }
+
+public string[] namesArray = ["Walter", "Skyler"];
```

Comment:
Shall we move this to inside the service?

Reply:
It was there to make a resource function non-isolated. Then the `invokeMethodAsyncSequentially()` method can be tested. If it was inside the service, I think the whole service is going to be non-isolated. Then the `invokeMethodAsyncConcurrently()` method will not run. To run both methods, array was kept there